shared/actions/config/index.js

```diff
@@ -33,6 +34,79 @@ function getConfig (): AsyncAction {
   }
 }
 
+let followers = {}
```

Comment:
Why is this data being stored in the actions? Shouldn't it only be obtained from the store via `getState`?